lib/components/ms-action-bar/MsActionBar.vue

```diff
@@ -3,13 +3,44 @@
 <template>
   <ion-item-divider class="toolbar ion-margin-bottom secondary">
     <div class="action-bar">
+      <ms-action-bar-button
+        v-for="(button, index) in buttons"
+        :key="index"
```

Comment:
Use the label as key instead of index, since the index can be the same for different button.